components/brave_new_tab_ui/stories/components.tsx

```diff
@@ -4,7 +4,7 @@
 
 import * as React from 'react'
 import { storiesOf } from '@storybook/react'
-import { withKnobs, select, text } from '@storybook/addon-knobs/react'
+import { withKnobs, select, text } from '@storybook/addon-knobs'
```

Comment:
note for reviewers: this was changed in storybook 5.2 (https://github.com/storybookjs/storybook/issues/8233#issuecomment-536976818)